lisa/tests/staging/sched_android.py

```diff
@@ -201,4 +201,90 @@ class UnboostedFreq(SchedTuneFreqBoost):
     """
     boost = 0
 
+
+class SchedTuneTaskPlacement(SchedTuneTest, abc.ABC):
+    """
+    Abstract class for testing SchedTune's task-placement biasing feature
```

Comment:
similar comment to another one: solely inheriting from abc.ABC does not make the class abstract (and in that case, it already inherits from it). Inheriting from abc.ABC *and* not implementing methods/attributes decorated with `@abc.abstractmethod` make a class abstract.